cet/shstk_alloc.c

```diff
@@ -28,6 +28,13 @@
 
 size_t shstk_size = 0x200000;
 
+#define get_ssp()						\
+({								\
+	unsigned long _ret;						\
```

Comment:
It should be better if the '\\' is aligned.

Reply:
Yes, done.

Thanks!